FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -256,13 +256,13 @@ extension MessageViewController {
 
     private func permanentlyDelete() {
         guard let input = input else { return hideSpinner() }
-        input.objMessage.flags = MCOMessageFlag(rawValue: input.objMessage.flags.rawValue | MCOMessageFlag.deleted.rawValue)
+        input.objMessage.flags = MCOMessageFlag(rawValue: MCOMessageFlag.deleted.rawValue)
```

Comment:
We're doing this two times - here and just below - that seams wrong. I would delete this line, what do you think?

Reply:
I removed this line because it's quite strange to have 
`input.objMessage.flags.rawValue | MCOMessageFlag.deleted.rawValue` so it means it will have original flag **OR** deleted flag, but leave in previous implementation to not break anything. 
I can take a look in this functionality, but would be nice to have some ACs for this. Can you please provide some "expected behaviour" for this?

Reply:
I think this is a binary (bitwise) OR. That's different than a logical OR. It's working on a bit by bit basis.

For example, if you have a bitmask for a one byte value:

```
first bit: delete
second bit: important
third bit: ..
```
and then you have an existing binary value (one byte), with various flags:
```
0100101
```
And the delete flag is
```
1000000
```
Then ORing them together produces
```
1100101
```
Meaning it changed the first bit representing the delete. This is just an example, look up the docs and also how bitwise or works.

That itself looks correct. What I meant is, that this line is here two times. Why change it two times?

![image](https://user-images.githubusercontent.com/6306961/86933008-97a1ba80-c129-11ea-8912-fbc73fa267db.png)

What's why I said the first changed line is probably there by error in the first place.